frontend/src/metabase/containers/Overworld.jsx

```diff
@@ -61,6 +61,11 @@ const getParitionedCollections = createSelector(
   },
 );
 
+const getGreeting = createSelector(
+  [getUser],
```

Comment:
Per [the docs](https://github.com/reduxjs/reselect#createselectorinputselectors--inputselectors-resultfunc), looks like making it an array isn't necessary